src/index.js

```diff
@@ -1,20 +1,5 @@
-import _ from 'lodash';
-import printMe from './print.js';
+// import _ from 'lodash';
 import './style.css';
```

Comment:
- [ ] Please consider removing commented codes as it will make your codebase messy, Remember if you ever like to see the code you have written once you can use your commit history.

Reply:
thank you for the feedback removed the comment

Reply:
Fixed.